scripts/core/auth/login-modal.html

```diff
@@ -23,12 +23,19 @@
                         </div>
                     </button>
                 </fieldset>
+
+                <fieldset class="buttons" ng-show="allowedAuth.google">
+                    <button class="btn btn--sd-green"
+                        ng-click="openLoginPopup('google')" translate>Log In with Google</button>
```

Comment:
vincers comment was about `Login` which was there before, `Log In` is updated version